app/assets/javascripts/account/token_copy.js

```diff
@@ -0,0 +1,15 @@
+$(document).ready(function () {
+  var inputEl = $("#deviceToken");
+  var btnEl = $("#copyBtn");
+
+  if (!inputEl || !btnEl) return;
+
+  inputEl.click(function() {
+    inputEl.select();
```

Comment:
what's the idea of clicking on an element and firing a change event ?
easier would be `$input.trigger("change")` or something like that